package.json

```diff
@@ -6,7 +6,7 @@
     "lint": "markdownlint README.md && npm run lint --workspaces",
     "prepare": "npm run build --workspaces",
     "start": "npm start --workspace=example",
-    "tag": "[ `git rev-parse --abbrev-ref HEAD` = 'main' ] && standard-version --no-verify",
+    "tag": "[ `git rev-parse --abbrev-ref HEAD` = 'main' ] && commit-and-tag-version --no-verify --npmPublishHint 'true'",
```

Comment:
TIL

> commit-and-tag-version is a fork of standard-version. Because of maintainer availability, standard-version was [deprecated on 15th May 2022](https://github.com/conventional-changelog/standard-version/pull/907). The previous maintainer recommends [release-please](https://github.com/googleapis/release-please) as an alternative for those who are using GitHub actions. This fork exists for those who can't switch to release-please, or who would like to continue using standard-version.


https://github.com/absolute-version/commit-and-tag-version?tab=readme-ov-file#commit-and-tag-version

💯